### PR TITLE
Add Infinite Tracing host to NR

### PR DESCRIPTION
**Pull request by nmujagic**

### CODE REVIEW

File: packages/kununu-utils/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kununu/kununu-utils",
-  "version": "2.2.3",
+  "version": "2.2.4",
```
Comment (codingcircus) [inline packages/kununu-utils/package.json:3]: I think it should be 2.3.0

**Review by codingcircus — CHANGES_REQUESTED** [review]

File: packages/kununu-utils/newrelic/getConfig/index.js [hunk 2]
```diff
@@ -35,6 +35,14 @@ module.exports = () => ({
   distributed_tracing: {
     enabled: 'true',
   },
+  /**
+   * Sets Infinite tracer host if available, see https://one.eu.newrelic.com/edge
+   */
+  infinite_tracing: {
+    trace_observer: {
+      host: (process.env.NR_INFINITE_TRACER_HOST) ? process.env.NR_INFINITE_TRACER_HOST : '',
```
Comment (Clasc) [inline packages/kununu-utils/newrelic/getConfig/index.js:43]: I think you could do this:
process.env.NR_INFINITE_TRACER_HOST ?? '',

would be a little cleaner :)

Comment (nmujagic) [reply packages/kununu-utils/newrelic/getConfig/index.js:43]: Reminder/Info: As seen, it breaks tests as its unsupported language features, so wee need to go with as is

**Review by codingcircus — APPROVED** [review]
lgtm

**Pull request merged**